### ISSUE TITLE
Suboptimal error

### ISSUE THREAD

**Issue by KAAtheWiseGit** [nushell/nushell#11018]
### Describe the bug

When deleting a folder owned by root as a user, for whom `trash` option for `rm` is enabled, the error message is a bit wordy:

```Nushell
/usr/share/fonts> rm -r monaspace/
Error: nu::shell::remove_not_possible

  × Remove not possible
   ╭─[entry #3:1:1]
 1 │ rm -r monaspace/
   ·       ─────┬────
   ·            ╰── Could not delete /usr/share/fonts/monaspace: FileSystem { path: "/home/kaathewise/.local/share/Trash/files/monaspace", kind: PermissionDenied }
Try '--trash' flag
   ╰────
```

### How to reproduce

1. Create an file/folder as root.
2. Enable `trash` `rm` option as another user.
3. Try to delete the root-owned object as the user.

### Expected behavior

A pretty-printed error.

### Configuration

| key                | value                                                                                                               |
| ------------------ | ------------------------------------------------------------------------------------------------------------------- |
| version            | 0.86.0                                                                                                              |
| branch             |                                                                                                                     |
| commit_hash        |                                                                                                                     |
| build_os           | linux-x86_64                                                                                                        |
| build_target       | x86_64-alpine-linux-musl                                                                                            |
| rust_version       | rustc 1.72.1 (d5c2e9c34 2023-09-13) (Alpine Linux 1.72.1-r0)                                                        |
| cargo_version      | cargo 1.72.1                                                                                                        |
| build_time         | 2023-10-22 21:58:37 +00:00                                                                                          |
| build_rust_channel | release                                                                                                             |
| allocator          | mimalloc                                                                                                            |
| features           | default, sqlite, trash, which, zip                                                                                  |
| installed_plugins  | custom-value generate, custom-value generate2, custom-value update, gstat, nu-example-1, nu-example-2, nu-example-3 |

**Comment by cptpiepmatz** [comment]
Is that fixed now? I forgot about this issue

**Comment by kaathewisegit** [comment]
yep, iirc this was happening somewhere in the middle of the migration to uutils
